<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientUI</class>
 <widget class="QMainWindow" name="ClientUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#ClientUI{	
	
	background-image: url(:/image/climain.jpg);

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(81, 81, 81);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhUppercaseOnly</set>
    </property>
    <property name="text">
     <string>校园快跑外卖订餐系统</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1061</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget_4{
	background-image: url(:/image/timg.jpg);
}</string>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>1031</width>
       <height>571</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
	
	background-color: rgb(255, 255, 255);
	
	
}</string>
     </property>
     <widget class="QPushButton" name="pb_clientmer">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>540</y>
        <width>76</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pb_clientmer{
	
	background-image: url(:/ico/pb1.jpg);
}
QPushButton#pb_clientmer{
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton#pb_clientmer:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton#pb_client:pressed {
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="Line" name="line">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1</width>
       <height>1</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>1031</width>
       <height>57</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="5">
       <widget class="QPushButton" name="pb_pswd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pb_pswd{
	
	background-image: url(:/ico/pb1.jpg);
}
QPushButton#pb_pswd{
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton#pb_pswd:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton#pb_pswd:pressed {
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
        </property>
        <property name="text">
         <string>修改密码</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="pb_shopcart">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pb_shopcart{
	
	background-image: url(:/ico/pb1.jpg);
}
QPushButton#pb_shopcart{
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton#pb_shopcart:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton#pb_form:pressed {
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
        </property>
        <property name="text">
         <string>购物小车</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pb_form">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pb_form{
	
	background-image: url(:/ico/pb1.jpg);
}
QPushButton#pb_form{
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton#pb_form:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton#pb_commodity:pressed {
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
        </property>
        <property name="text">
         <string>订单管理</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pb_merchant">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pb_merchant{
	
	background-image: url(:/ico/pb1.jpg);
}
QPushButton#pb_merchant{
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton#pb_merchant:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton#pb_shop:pressed {
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
        </property>
        <property name="text">
         <string>商家管理</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pb_first">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pb_first{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton#pb_first:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton#pb_first:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
        </property>
        <property name="text">
         <string>用户首页</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pb_mydata">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pb_mydata{
	
	background-image: url(:/ico/pb1.jpg);
}
QPushButton#pb_mydata{
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton#pb_mydata:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}



QPushButton#pb_mydata:pressed {
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}
</string>
        </property>
        <property name="text">
         <string>我的资料</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>630</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lb_time">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>630</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="pb_f">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
        </property>
        <property name="text">
         <string>首页</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pb_go">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>20</pointsize>
      <strikeout>true</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="maximumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFirst"/>
   <addaction name="actionMerchant"/>
   <addaction name="actionForm"/>
   <addaction name="actionShopcart"/>
   <addaction name="actionTalk"/>
   <addaction name="actionMydata"/>
   <addaction name="actionUpdatepswd"/>
  </widget>
  <action name="actionFirst">
   <property name="text">
    <string>first</string>
   </property>
  </action>
  <action name="actionMydata">
   <property name="text">
    <string>mydata</string>
   </property>
  </action>
  <action name="actionUpdatepswd">
   <property name="text">
    <string>updatepswd</string>
   </property>
  </action>
  <action name="actionMerchant">
   <property name="text">
    <string>merchant</string>
   </property>
  </action>
  <action name="actionForm">
   <property name="text">
    <string>form</string>
   </property>
  </action>
  <action name="actionShopcart">
   <property name="text">
    <string>shopcart</string>
   </property>
  </action>
  <action name="actionTalk">
   <property name="text">
    <string>talk</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
